### ISSUE TITLE
Agent scripts not working?

### ISSUE THREAD

**Issue by kareltucek** [UltimateHackingKeyboard/agent#1905]
Three problems (should I split them into different tickets?):

- [x] scripts are not working when build from agent root (`agent$ rm -rf node_modules/ && npm ci && npm run build`):

```
agent/packages/usb$ ./get-module-state.ts
(node:2021066) ExperimentalWarning: Custom ESM Loaders is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
Error [ERR_LOADER_CHAIN_INCOMPLETE]: "file:///opt/agent/node_modules/ts-node/child-loader.mjs 'resolve'" did not call the next hook in its chain and did not explicitly signal a short circuit. If this is intentional, include `shortCircuit: true` in the hook's return.
    at new NodeError (node:internal/errors:387:5)
    at ESMLoader.resolve (node:internal/modules/esm/loader:852:13)
    at async ESMLoader.getModuleJob (node:internal/modules/esm/loader:431:7)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:533:24)
    at async loadESM (node:internal/process/esm_loader:91:5)
    at async handleMainPromise (node:internal/modules/run_main:65:12) {
  code: 'ERR_LOADER_CHAIN_INCOMPLETE'
}
```

Furthermore:

- [x] packages/usb/README.md instructs to run `npm install` in the packages/usb directory. This results in:

```
agent/packages/usb$ npm i
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/kboot - Not found
npm ERR! 404
npm ERR! 404  'kboot@0.0.0' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/me/.npm/_logs/2022-11-10T09_56_05_565Z-debug-0.log
```

This is with:
```
$ node --version
v16.17.0
```

- [x] root README.md instructs to use node 12. However, this results in:
```
$ sudo n 12
   installed : v12.22.12 (with npm 6.14.16)
$ rm -rf node_modules/ && npm ci && npm run build

> phantomjs-prebuilt@2.1.16 install /opt/agent/node_modules/phantomjs-prebuilt
> node install.js

PhantomJS not found on PATH
Download already available at /tmp/phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2
Verified checksum of previously downloaded file
Extracting tar contents (via spawned process)
Removing /opt/agent/node_modules/phantomjs-prebuilt/lib/phantom
Copying extracted folder /tmp/phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2-extract-1668074738064/phantomjs-2.1.1-linux-x86_64 -> /opt/agent/node_modules/phantomjs-prebuilt/lib/phantom
Writing location.js file
Done. Phantomjs binary available at /opt/agent/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs

> node-hid@2.1.2 install /opt/agent/node_modules/node-hid
> prebuild-install --runtime napi || node-gyp rebuild


> @parcel/watcher@2.0.4 install /opt/agent/node_modules/@parcel/watcher
> node-gyp-build


> nx@15.0.4 postinstall /opt/agent/node_modules/nx
> node ./bin/compute-project-graph


> electron@21.2.0 postinstall /opt/agent/node_modules/electron
> node install.js


> highlight.js@9.18.5 postinstall /opt/agent/node_modules/highlight.js
> node deprecated.js

-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

  Verion 9 of Highlight.js has reached EOL.  It will no longer
  be supported or receive security updates in the future.
  Please upgrade to version 10 or encourage your indirect
  dependencies to do so.

  For more info:

  https://github.com/highlightjs/highlight.js/issues/2877
  https://github.com/highlightjs/highlight.js/blob/master/VERSION_10_UPGRADE.md

-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

> core-js@3.26.0 postinstall /opt/agent/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js:
> https://opencollective.com/core-js
> https://patreon.com/zloirock
> bitcoin: bc1qlea7544qtsmj2rayg0lthvza9fau63ux0fstcz

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)


> uhk-agent@2.0.1 postinstall /opt/agent
> lerna bootstrap

/opt/agent/node_modules/@lerna/package-graph/index.js:79
            if (depNode?.version) {
                        ^

SyntaxError: Unexpected token '.'
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/opt/agent/node_modules/@lerna/query-graph/query-graph.js:3:26)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! uhk-agent@2.0.1 postinstall: `lerna bootstrap`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the uhk-agent@2.0.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/me/.npm/_logs/2022-11-10T10_05_47_877Z-debug.log
```

**Comment by mondalaci** [comment]
Thanks for reporting these issues! No need to split this issue.

**Comment by ert78gb** [comment]
@kareltucek I think the problem is the outdated documentation. I updated both documentations.

**Comment by mondalaci** [comment]
Should be resolved by the recent readme updates.

**Comment by kareltucek** [comment]
Yes! Solved by downgrading to 16.16 from 16.17.
